src/applications/validate-mhv-account/containers/ErrorMessage.jsx

```diff
@@ -14,10 +9,13 @@ import CreateAccountFailed from '../components/errors/CreateAccountFailed';
 import UpgradeAccountFailed from '../components/errors/UpgradeAccountFailed';
 import { ACCOUNT_STATES } from './../constants';
 
-class ErrorMessage extends React.Component {
+export default class ErrorMessage extends React.Component {
```

Comment:
"Disconnected" this component, and moved `upgradeMHVAccount()` into `<UpgradeAccountFailed/>`